src/app/modules/inbox/inbox.component.ts

```diff
@@ -72,7 +73,7 @@ export class InboxComponent implements OnInit {
 
     reject() {
         this.selected = {
-            'tag_id': '1',
+            'tag_id': '2',
```

Comment:
getting error "A class member cannot have the 'const' keyword". so i have used readonly